string room_name
---
bool destination_reached